src/css/font.ts

```diff
@@ -0,0 +1,18 @@
+import { css } from 'styled-components/macro'
+
+export const globalFontStyles = css`
```

Comment:
nit: these are the only font styles, they can just be called `font` or `fontStyles`.

Reply:
for imported font styles i like to specify that these are indeed the global font styles to help discern if they're, like, specific to this component or not